src/application.coffee

```diff
@@ -132,7 +133,7 @@ logSpecialAction = (action, value, success) ->
 		msg = "Applying config variable #{action} = #{value}"
 	logSystemMessage(msg, {}, "Apply special action #{if success then "success" else "in progress"}")
 
-application.kill = kill = (app, updateDB = true, removeContainer = true) ->
+application.kill = kill = (app, { updateDB = true, removeContainer = true }) ->
```

Comment:
You still need the `= {}` ie `{ updateDB = true, removeContainer = true } = {}` otherwise it will throw when trying to dereference a null/undefined opts arguments